.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+### What is the feature/fix?
+
+** Describe the task and what is the goal for it. **
+** Describe the bug and the solution. **
+** Any additional information **
+
```

Comment:
should add a line to add screenshot or video or note on the feature if have any?